UnityProject/Assets/Scripts/UI/Objects/Medical/MedicalCon/GUI_MedicalConsoleEntry.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Items;
+using Objects.Medical;
+using UI.Core.NetUI;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI.Objects.Medical.MedicalCon
+{
+	public class GUI_MedicalConsoleEntry : DynamicEntry
+	{
+		[SerializeField] private NetText_label label;
+		[SerializeField] private Image image;
+		[SerializeField] private Color unknownColor;
+		[SerializeField] private Color healthyColor;
+		[SerializeField] private Color dangerColor;
+		[SerializeField] private Color deadColor;
+
+		public void SetValues(MedicalTerminal.HealthInfo info)
+		{
+			label.MasterSetValue($"{info.Info}");
+			if (info.Mode == SuitSensor.SensorMode.OFF)
+			{
+				image.color = unknownColor;
```

Comment:
This won't network?

Reply:
i know, but eh, what do i do?

Reply:
you make the colour changing component synchronised thingy, be able to specify the component it changes the colour for, and Then add an extra layer and put the text and the colour on each layer separately

Reply:
what color changing component thingy
what extra layer and how

Reply:
is there anything for actually networking Colour currently?

Reply:
nope

Reply:
I've decided to just not add background color changes as it ruins visibility